### PR TITLE
Support for ENUM type

### PR DESCRIPTION
**Pull request by suranjan**
With this change the ENUM type is sent as STRING and values are also sent as STRING. There is a test, however that doesn't work because the docker image doesn't have yugabytedb version that supports ENUM type yet

### CODE REVIEW

File: src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java [hunk 1]
```diff
@@ -220,6 +228,31 @@ public void testVerifyValue() throws Exception {
...
+
+        // 3 because there are 3 enum values in the enum type
+        final long recordsCount = 3;
+
+        awaitUntilConnectorIsReady();
+
+        // 3 records will be inserted in the table test_enum
+        insertEnumRecords();
+
+        CompletableFuture.runAsync(() -> verifyPrimaryKeyOnly(recordsCount))
```
Comment (vrajat) [inline src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java:250]: Is there an assert to ensure enum value is correct in the source record?

**Review by vrajat — COMMENTED** [review]
Does the test check for enum values?

File: src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java (hunk 1 above)
Comment (vaibhav-yb) [reply src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java:250]: We can add one anytime.

**Review by vaibhav-yb — APPROVED** [review]
Looks good to me. The test can be changed to use a DB version with ENUM support in the next iteration.

**Comment by suranjan** [comment]
> Does the test check for enum values?

yes, added the test.

**Pull request merged**